awx/ui_next/src/screens/Template/JobTemplateAdd/JobTemplateAdd.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  PageSection,
+  Tooltip,
+} from '@patternfly/react-core';
+import CardCloseButton from '@components/CardCloseButton';
+import JobTemplateForm from '../shared/JobTemplateForm';
+import { JobTemplatesAPI } from '@api';
+
+function JobTemplateAdd({ history, i18n }) {
+  const [error, setError] = useState('');
+
+  const handleSubmit = async values => {
+    try {
+      const data = await JobTemplatesAPI.create(values);
+      const { response } = data;
+      history.push(`/templates/${response.type}/${response.id}/details`);
+    } catch (err) {
+      setError(err);
+    }
+  };
```

Comment:
I'm seeing an error without redirect to the `details` page on successful creation requests: 
![4257](https://user-images.githubusercontent.com/9753817/60607377-26e3f780-9d8b-11e9-9e38-704b57470afd.gif)

Reply:
see https://github.com/ansible/awx/pull/4257/files#r300046553